tests/playwright-tests/general-public/pages/glossary-page.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable lines-between-class-members */
+import { Locator, Page } from '@playwright/test';
+
+// Glossary page
+export class GlossaryPage {
+  readonly page: Page;
+  readonly page_search_box: Locator;
```

Comment:
I don't know if there are established variable name styles that are specific to Playwright or if there's a convention of snakecase variables where you've worked before?

If not, we tend to use camelCase variable names in Typescript projects.

So for instance, `page_search_box` would be `pageSearchBox` instead.

Reply:
Sure I will update that accordingly